### PR TITLE
Fix extraneous whitespace in check output

### PR DESCRIPTION
**Pull request by wvu**
Death to the peer gods.

```
msf5 exploit(unix/local/exim_perl_startup) > check

[!] SESSION may not be compatible with this module.
[*]  The target is not exploitable.
msf5 exploit(unix/local/exim_perl_startup) >
```

#6526, #6907

### CODE REVIEW

File: lib/msf/ui/console/module_command_dispatcher.rb [hunk 1]
```diff
@@ -221,6 +221,7 @@ def check_simple(instance=nil)
       rport = instance.rport if instance.respond_to?(:rport)
       peer = "#{rhost}:#{rport}"
     end
+    peer_msg = "#{peer} - " if peer
```
[the remark this answers is not in the document]
Comment (bcoles) [reply lib/msf/ui/console/module_command_dispatcher.rb]: It's fine. Gets resolved to `nil`, which is also fine.

```
# irb
2.3.0 :001 > peer = nil
 => nil 
2.3.0 :002 > peer_msg = "#{peer} - " if peer
 => nil 
2.3.0 :003 > puts peer_msg

 => nil 
2.3.0 :004 > 
```

[the remark this answers is not in the document]
Comment (wvu-r7) [reply lib/msf/ui/console/module_command_dispatcher.rb]: The interpolation that uses it turns it into an empty string. Let's play it safe, though.

[the remark this answers is not in the document]
Comment (wvu-r7) [reply lib/msf/ui/console/module_command_dispatcher.rb]: I tested it, and it was fine, but I'm going ahead with your change for safety. I wasn't thinking too hard on this one. Thanks!

[the remark this answers is not in the document]
Comment (wvu-r7) [reply lib/msf/ui/console/module_command_dispatcher.rb]: (Someone may use it later and do something silly like concatenate.)

**Comment by busterb** [comment]
## Release Notes

This fixes a formatting error when printing messages while running the `check` command from local exploit modules.

**Comment by wvu-r7** [comment]
Thanks, @busterb && @bcoles!

**Pull request merged**